webapp/public/locales/en/main.json

```diff
@@ -644,6 +644,7 @@
   "xpansion.amplPresolve": "Ampl presolve",
   "xpansion.amplSolverBoundsFrequency": "Ampl solve bounds frequency",
   "xpansion.relativeGap": "Relative gap",
+  "xpansion.batchSize": "batch size",
```

Comment:
For consistence, use upper case for the first letter : `Batch size`

Reply:
done